Format code also adding required attribute to those fields that need it.

anthonythorne opened the pull request:
Add basic form field validation, required, and number ranges.
Also formatted the code in the forms directory to be in line with wpphpcs nags.
Updated readme to elaborate on what fields are available on some of the filters.

https://github.com/TheCodeCompany/wp-chargify/issues/114

anthonythorne committed:
Format code also adding required attribute to those fields that need it.

anthonythorne committed:
Merge branch 'master' into 114-basic-validation-html5

BronsonQuick commented on plugins/wp-chargify/includes/forms/signup.php:
```diff
@@ -8,43 +16,106 @@
+			'save_fields'  => false,
+			'autocomplete' => 'on',
+			'attributes'   => [
+				'class'        => 'test',
```
Class `test` seems like a strange class name?

anthonythorne replied:
Yep it does....

Removed it.

BronsonQuick commented on plugins/wp-chargify/includes/forms/signup.php:
```diff
@@ -8,43 +16,106 @@
+			'object_types' => [ 'chargify_account' ],
+			'hookup'       => false,
+			'save_fields'  => false,
+			'autocomplete' => 'on',
```
Good catch on the missing `autocomplete`!

anthonythorne replied:
this actually didn't work, I followed the cmb code back to 'form_format' where it was hardcoded, hence the form_format() function extending the hard code and allowing attributes, and or class name/extension from the original. Which came in handy during theme implementation.

I have removed the above line, which was remnant from testing.

BronsonQuick commented on plugins/wp-chargify/readme.txt:
```diff
@@ -49,16 +49,65 @@ We provide a number of optional filters you can use:
+
 `chargify_customer_details_fields` - Filter the Customer Details form field. Use `$signup_form->add_field` and `$signup_form->remove_field` to add and/or remove fields.
+	Available fields by id;
+		'chargify_customer_details_title'
```
There's an extra tab here

BronsonQuick commented on plugins/wp-chargify/readme.txt:
```diff
@@ -49,16 +49,65 @@ We provide a number of optional filters you can use:
+
 `chargify_account_details_fields` - Filter the Account Details form field. Use `$signup_form->add_field` and `$signup_form->remove_field` to add and/or remove fields.
+	Available fields by id;
+		'chargify_account_details_title'
```
Another extra tab here.

BronsonQuick reviewed (changes requested):
Awesome work dude! Just a couple of minor things. Thanks for fixing up some of the formatting too. I will have to check my sniffs on this new laptop as I don't think they are configured correctly yet.

anthonythorne committed:
Readme format update

anthonythorne committed:
Remove test code.

anthonythorne committed:
Remove the autocomplete that is remnant from testing, this did not work hence the form_format() function.

BronsonQuick reviewed (approved):
Looks good to me. Nice one!
